REQUIRE

```diff
@@ -7,3 +7,4 @@ SortingAlgorithms
 Reexport
 Compat 0.19.0
 FileIO 0.1.2
+CSV
```

Comment:
If we don't add deprecations, we don't need to depend on CSV.jl at all. Just add it to `test/REQUIRE`, and add `using CSV` to places that rely on it. Some places of the manual may need a `Pkg.add("CSV")` mention for newcomers.